apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -105,9 +105,8 @@ export default class LessonEditor extends Component {
             />
             <HelpTip>
               <p>
-                Check this box if this lesson should be locked from teachers.
-                Only validated teachers will be able to see it and unlock the
-                materials.
+                Check this box if this lesson should be locked for students and
+                teachers can unlock it for their students.
```

Comment:
This was a bit confusing to me to read, because I had to think about whether I was controlling whether it would be locked for students, or unlockable by teachers. How does this sound?

"Check this box if this lesson should be locked for students. If checked, teachers will be able to unlock the lesson for their students."

Feel free to pick another wording or ignore if that does not sound like an improvement.

Reply:
I didn't love what I had so this sounds good me me. Will update